src/masternode.cpp

```diff
@@ -293,6 +294,13 @@ bool CMasternodeBroadcast::Create(const std::string& strService,
     if (!CheckDefaultPort(_service, strErrorRet, "CMasternodeBroadcast::Create"))
         return false;
 
+    // Check if the MN has a ADDRv2 and reject it if the new NU wasn't enforced.
+    if (!_service.IsAddrV1Compatible() &&
+        !Params().GetConsensus().NetworkUpgradeActive(chainHeight, Consensus::UPGRADE_V5_3)) {
+        strErrorRet = "Cannot start MN with a v2 address before the v5.3 enforcement\n. Let the node fully sync before start the masternode";
+        return false;
+    }
```

Comment:
Nit: The second line (`Let the node fully sync before start the masternode`) results cropped and hard to read in the GUI `SnackBar`. 
It is also redundant, as the block-sync is already checked for at line 266, so the node is synced here. So I think we can just drop it (besides, it should be "before start**ing**" imo).

Reply:
sure, pushed.